src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -181,27 +181,44 @@ def get_paths_to_platform_and_hwsku_dirs():
 
     return (platform_path, hwsku_path)
 
-
-def get_path_to_port_config_file():
+def get_path_to_port_config_file(asic=None):
     """
     Retrieves the path to the device's port configuration file
 
     Returns:
         A string containing the path the the device's port configuration file
     """
-    # Get platform and hwsku path
+
+    '''
+    This platform check is performed to make sure we return a None
+    in case of unit-tests within sonic-cfggen where platform is not expected to be
+    present because tests are not run on actual Hardware/Container.
+    This check should however be ommitted once Sonic-cfggen does not use
+    port_config file for unit-tests.
```

Comment:
This part of the comment should read:

```
TODO: refactor sonic-cfggen such that we can remove this check
```

Also please either use double-quotes for the docstring to align with the rest of the file, or use `#` for these comments, as they are not as much documentation as they are just informative comments.

Reply:
fixed both